core/src/main/java/org/apache/accumulo/fate/ZooStore.java

```diff
@@ -111,6 +111,11 @@ public ZooStore(String path, ZooReaderWriter zk) throws KeeperException, Interru
     zk.putPersistentData(path, new byte[0], NodeExistsPolicy.SKIP);
   }
 
+  /**
+   * For testing only
+   */
+  public ZooStore() {}
+
```

Comment:
This can be package-private if the tests that use it are all unit tests in the same package.